presto-orc/src/main/java/com/facebook/presto/orc/ColumnGroupReader.java

```diff
@@ -66,6 +67,7 @@
     private int[] filterResults;
     private Map<Integer, StreamReader> channelToStreamReader;
     private int[] outputChannels;
+    private final Block[] constantBlocks;
```

Comment:
It would be helpful to group immutable variables and separate these from mutable state. Let's put constantBlocks,  outputChannels and filterFunctions together at the beginning of the class before any mutable variable and add an empty line to separate immutable and mutable variables.

```
    private static final int BUDGET_HARD_LIMIT_MULTIPLIER = 4;
    private static final int MIN_READER_BUDGET = 8000;

    private final FilterFunction[] filterFunctions;
    private final int[] outputChannels;
    private final Block[] constantBlocks;

    // Number of complete rows in result Blocks in StreamReaders.
    private int numRowsInResult;
    ....
```